test/extended/cluster/README.md

```diff
@@ -0,0 +1,46 @@
+# Cluster Mirror
+
+This document describes how to use the Cluster Mirror to create a Cluster
```

Comment:
Start with an intro to why the tool was written:

The cluster mirror tool was conceived to provide a way to reproduce (with reasonably approximate fidelity) a control-plane workload.  The idea is to make sure our scale testing efforts are as realistic as possible.

Use-cases include:
* Replicating OpenShift Online clusters into lab environments for scale testing and R&D.
* Reproducing customer environments in support situations.

It is important to note that this tool does not inspect persistent storage, it does not look at the content of secrets and it does not export user data in any way.  It takes a "fingerprint" of the target environment via API calls, and allows users to "replay" that environment elsewhere (in terms of number of namespaces, templates and pods and what type they are).